brightlayer-ui/colors
# move licenses file & include with builds

Fixes # .


#### Changes proposed in this Pull Request:
- Move license file from root to each dist folder
- Add additional file tests
- Add file name to files list in package.json


#### Screenshots:


#### To Test:
- 
open branch and remove the licenses.json from both the branding and UI folders.
Run yarn precommit
Licenses.json should be created and place back in Branding/UI folders
Test should reflect that the licenses are found
